modules/installation-configure-proxy.adoc

```diff
@@ -62,6 +62,19 @@ range that is specified in the `networking.machineNetwork[].cidr` field in the
 ====
 endif::bare-metal[]
 
+ifdef::openstack[]
+[NOTE]
+====
+For Kuryr installations running on Restricted Network and using Proxy,
+the host that has proxy configured needs to have a route entry specifying
+which host interface should be used to handle the Pods Network traffic.
+
+`sudo ip route add <CLUSTER_NETWORK_CIDR> via <INSTALLER_SUBNET_GATEWAY>`
```

Comment:
I think in corporate environments the proxy will not run on bastion VM but will be some service managed by the corporate IT. So I'd make this more generic - simply the networking setup has to allow proxy to reply into the OpenShift cluster's router.